package.json

```diff
@@ -44,7 +44,9 @@
     "build-storybook": "build-storybook",
     "release": "yarn build && auto shipit",
     "test-storybook:jsdom": "jest --no-cache --config ./jsdom-jest.config.js -i",
-    "test-storybook:playwright": "jest --no-cache --config ./test-runner-jest.config.js"
+    "test-storybook:playwright-no-cache": "jest --no-cache --config ./test-runner-jest.config.js",
+    "test-storybook:playwright": "./bin/test-storybook.js",
+    "test-storybook:playwright-json": "./bin/test-storybook.js --stories-json"
```

Comment:
When running this command locally I am seeing a weird behavior where the stories are running in permutation:

![image](https://user-images.githubusercontent.com/1671563/151351390-f3d721c7-5aee-4785-9eee-da0128dc8a07.png)

Reply:
When running with `TARGET_URL=https://mealdrop.vercel.app/storybook yarn test-storybook:playwright-json` my tests are hanging for the entire time the test will take in this screen:

![image](https://user-images.githubusercontent.com/1671563/151352183-2d4a0c49-b843-4b49-8087-d47bbcb3bdcd.png)

So there's no feedback per test and people might think it's hanging as a bug? Is it possible to create multiple test suites instead?

After it successfully runs, I see:
```
Test Suites: 1 passed, 1 total
Tests:       3520 passed, 3520 total
```

It's also related to the permutation I mentioned above:

![image](https://user-images.githubusercontent.com/1671563/151352360-a6177907-4505-44d4-bbe8-2da93242cd87.png)